frontend/packages/insights-plugin/src/components/InsightsPopup/index.tsx

```diff
@@ -68,9 +75,31 @@ export const InsightsPopup: React.FC<PrometheusHealthPopupProps> = ({ responses,
     critical: 'insights-plugin~critical',
   };
 
+  const lastRefreshTime = parseInt(lastGatherResponse?.data?.result?.[0]?.value?.[1] || '0', 10);
+  // Prometheus generates time in ms, not ns
+  const relative = fromNow(new Date(lastRefreshTime * 1000));
+
   return (
     <Stack hasGutter>
       <StackItem>
+        {t('insights-plugin~Last refresh')}:{' '}
+        <span>
+          {!lastGatherError && lastRefreshTime ? (
+            <Tooltip
+              content={
+                <span>
+                  <Timestamp timestamp={lastRefreshTime} isUnix simple /> UTC
+                </span>
+              }
+            >
+              <span>{relative[0].toLowerCase() + relative.substring(1)}</span>
+            </Tooltip>
+          ) : (
+            t('insights-plugin~unknown')
+          )}
```

Comment:
@spadgett, yep, indeed. However, it was requested by UX to display "unknown" text, not "-" which is rendered by `Timestamp` in error/invalid cases. Do you want to keep it as it is in this patch, or do we have to modify the Timestamp so that it returns "unknown" text instead "-"?

Reply:
We don't want to change the Timestamp behavior.

But doesn't Timestamp show the popover for UTC time already? We shouldn't need to reimplement that, and we should strive to keep the display consistent with the rest of console. I'd argue `-` is better than `unknown` for that reason.

Reply:
@spadgett, oh, I didn't notice the Timestamp already renders a popover, thank you. Well, the initial proposal was to have a relative time (e.g., 2 hours ago), however, Timestamp doesn't have this option implemented, thus I needed to go a different path. However, our UX folks are OK with this Console timestamp format and I changed it according to your proposal. PTAL (also the screenshots are updated)

Reply:
@gkarat Timestamp does show relative times if the time is less than ~10 minutes, otherwise it shows the full time. (This would be consistent with the rest of console.)

Reply:
@spadgett, ok, this works for us (even though we initially wanted to **always** show the relative time). Might be worth enhancing the Timestamp and parameterizing this behavior in the future.